fish-chips.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  <a href="index.html">HOME</a>
+  <h1>FISH AND CHIPS RECIPE</h1>
+  <img src="images/fishchips.jpeg" alt="a picture of fish and chips">
+     <h2>INGREDIENTS</h2>
+    <ul>
+<li>7 tablespoons (55 grams) all-purpose flour, divided</li>
+  <li>7 tablespoons (55 grams) cornstarch</li>
+  <li>1 teaspoon baking powder</li>
+  <li>Sea salt, to taste</ls>
+  <li>1 pinch freshly ground black pepper, to taste</li>
+  <li>1/3 cup dark beer,cold</li>
+  </ul>
+  </body>
+  <body><h3>PREPARATION TIME</h3>
+    <ul>
+  <li>Prep:20 mins</li>
+<li>Cook:35 mins</li>
+<li>Rest Time :30 mins</li>
+<li>Total:85 mins</li> 
+</ul>
+<body>
```

Comment:
Here you are starting another body tag within the previous one. As stated above, only one body tag necessary :)!